apps/src/gamelab/GameLab.js

```diff
@@ -366,6 +370,14 @@ GameLab.prototype.onIsDebuggingSpritesChange = function (isDebuggingSprites) {
   this.gameLabP5.debugSprites(isDebuggingSprites);
 };
 
+GameLab.prototype.onStepSpeedChange = function (stepSpeed) {
+  this.gameLabP5.changeStepSpeed(stepSpeed);
+
+  if (this.pauseTickTimer()) {
+    this.startTickTimer();
```

Comment:
This took me a while to interpret.  A couple of things that might make this easier to read:

1. `pauseTickTimer` isn't really an accurate description - `stopTickTimer` or `clearTickTimer` might be clearer.
2. Documenting the return value of `pauseTickTimer` would help - and it's a little surprising that method has a return value at all.
3. Doesn't this basically vary on whether we're currently running or not?  Might read better:
   ```js
   if (this.isRunning()) {
     this.clearTickTimer();
     this.startTickTimer();
   }
   ```

Reply:
👍 
Renamed function
Removed return value
Added `isTickTimerRunning()` function
Changed to:
```
  if (this.isTickTimerRunning()) {
    this.stopTickTimer();
    this.startTickTimer();
  }
```